packages/jsii-java-runtime/project/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
what is this?

Reply:
This is so I can use `Mockito.mock` on `final` classes.